### PR TITLE
feat(actions): moving with J and K on select mode

### PR DESCRIPTION
**Pull request by WittBulter**
Add `J` & `K` to move action, only work on `select` mode. (this is similar to `vim`, friendship and quickness.)

### CODE REVIEW

**Comment by WittBulter** [comment]
@terkelg please help me to review this pr.

File: lib/elements/prompt.js [hunk 1]
```diff
@@ -19,14 +19,17 @@ class Prompt extends EventEmitter {
     this.in = opts.in || process.stdin;
     this.out = opts.out || process.stdout;
     this.onRender = (opts.onRender || (() => void 0)).bind(this);
+    
+    const className = this.constructor.name.toLowerCase();
+    this.isSelect = className === 'selectprompt' || className === 'multiselectprompt';
```
Comment (lumio) [inline lib/elements/prompt.js]: You could just use a local variable and do something like

```js
const isSelect = ['SelectPrompt', 'MultiselectPrompt'].indexOf(this.constructor.name) > -1;
```

File: lib/elements/prompt.js [hunk 2]
```diff
@@ -19,14 +19,17 @@ class Prompt extends EventEmitter {
...
 
     const rl = readline.createInterface(this.in);
     readline.emitKeypressEvents(this.in, rl);
 
     if (this.in.isTTY) this.in.setRawMode(true);
 
     const keypress = (str, key) => {
-      let a = action(key);
+      let a = action(key, this.isSelect);
```
Comment (lumio) [inline lib/elements/prompt.js]: As said before, a local variable should be enough :)

**Comment by terkelg** [comment]
@lumio feel free to merge this one when you're happy with the PR. 👍

**Comment by lumio** [comment]
Awesome thing! Love vim myself, so ;) Thanks @WittBulter

**Pull request merged**